### ISSUE TITLE
'az ml computetarget attach aks' should support "internal" LoadBalancer

### ISSUE THREAD

**Issue by pranaysahith** [Azure/azure-cli-extensions#2056]
### Extension name (the extension in question)
azure-cli-ml

### Description of issue (in as much detail as possible)
Currently `az ml computetarget update aks` command supports to update LoadBalancer to Internal type but this commands needs the AKS cluster to be already attached to the AML. 
It will be useful to support the `[--load-balancer-type]` argument to `az ml computetarget attach aks` command so that internal LoadBalancer type can be created when AKS is attached to AML.

-----

**Comment by yonzhan** [comment]
ml

**Comment by SureshBettadapur** [comment]
yes, required feature. Any update on when this can be made available?
